### DISCUSSION TITLE
google.maps TS2304: 'Cannot find name ...'

### DISCUSSION THREAD

**Discussion by itsandreramon** [DefinitelyTyped/DefinitelyTyped#54582] [Issues with a @types package]
When trying to compile an application that imports types from google.maps, I get the following errors when opening the file. I cannot compile my project. I am using all the latest versions.

<img width="787" alt="Screenshot 2021-07-16 at 23 09 12" src="https://user-images.githubusercontent.com/17139385/126008760-fa2b54a5-3b43-4389-bef9-128c7d547a64.png">

**Comment by jpoehnelt** [comment]
Your tsconfig needs browser types using `DOM` in the https://www.typescriptlang.org/tsconfig#lib config.
